pom.xml

```diff
@@ -87,6 +87,15 @@
     <url>https://github.com/quarkusio/quarkus/issues</url>
   </issueManagement>
 
+  <developers>
+    <developer>
+      <id>quarkiverse</id>
+      <name>Quarkiverse Community</name>
```

Comment:
I'd say it's still "Quarkus community".

Reply:
And organization and organizationUrl is Quarkus and https://quarkus.io respectively?